src/core/server/saved_objects/import/import_saved_objects.test.ts

```diff
@@ -89,8 +97,12 @@ describe('#importSavedObjectsFromStream', () => {
   let savedObjectsClient: jest.Mocked<SavedObjectsClientContract>;
   let typeRegistry: jest.Mocked<ISavedObjectTypeRegistry>;
   const namespace = 'some-namespace';
+  const testDataSourceId = 'some-datasource';
 
-  const setupOptions = (createNewCopies: boolean = false): SavedObjectsImportOptions => {
+  const setupOptions = (
+    createNewCopies: boolean = false,
+    dataSourceId: string | undefined
```

Comment:
can we give `undefined` as default value for `dataSourceId` parameter here? You won't need to change setOptions call in following code.

Reply:
Thanks for the suggestion, already updated